internal/proxy/github.go

```diff
@@ -184,14 +190,12 @@ func (p *Proxy) ProcessBuffer() {
 									}
 								}
 								break
-							} else if resp.StatusCode == 404 || resp.StatusCode == 400 { //400 - missing payload
-								log.Warn(fmt.Sprintf("Got status %s after retrying request on %s, throwing away the request", resp.Status, req.URL))
 							}
 
-							log.WithField("ns", ns).Infof(fmt.Sprintf("Request for %s to %s forwarded.", ns, req.Host))
 						}
 
-						//Delete request if we tried too many times or the replay was successful
+						// Deleting request since we tried too many times or the replay was successful
```

Comment:
wouldn't this part be executed even if r.Retries < Max ?

Reply:
Yes, but it is desirable. It is out of the if block. This part would be execute only in case of 
```
if  (r.Retries < Max){
     if (we find 200, 404, 404 ){
         execute delete;
}
} else {
        execute delete;
}
```
The actual logic is written in a different way